libs/core/web/src/lib/pages/login/Login.tsx

```diff
@@ -54,60 +76,70 @@ export function Login() {
-                    size={EnumSize.small}
-                    error={errors.email?.message}
-                    id="email"
-                    type="email"
-                    label="Email *"
-                    variant={EnumVariant.outlined}
-                    register={register}
-                  />
-                </Grid>
-                <Grid item>
-                  <InputTextField
-                    size={EnumSize.small}
-                    error={errors.password?.message}
-                    id="password"
-                    type="password"
-                    label="Password *"
-                    variant={EnumVariant.outlined}
-                    register={register}
-                  />
-                </Grid>
-                <Grid item>
-                  <Button variant="outlined" type="submit" fullWidth>
-                    Connexion
-                  </Button>
-                </Grid>
-              </Grid>
-            </Box>
+        <Box
+          component="form"
+          sx={{
+            minWidth: '20rem',
+          }}
+        >
+          <Grid container direction="column" spacing={1}>
+            <Grid item>
+              <InputTextField
+                size={EnumSize.small}
+                error={
+                  errors?.email?.message ? t(`${errors?.email?.message}`) : ''
+                }
+                id="email"
+                type="email"
+                label={`${t('email')} *`}
+                variant={EnumVariant.outlined}
+                register={register}
+              />
+            </Grid>
+            <Grid item>
+              <InputTextField
+                size={EnumSize.small}
+                error={
+                  errors?.password?.message
+                    ? t(`${errors?.password?.message}`)
+                    : ''
+                }
+                id="password"
+                type="password"
+                label={`${t('password')} *`}
+                variant={EnumVariant.outlined}
+                register={register}
+              />
+            </Grid>
+            <Grid item>
+              <LoadingButton
+                variant="outlined"
+                type="submit"
+                fullWidth
+                onClick={handleSubmit(onSubmit)}
+                loading={loading}
+                endIcon={<LockOpenIcon />}
+                loadingPosition="end"
+              >
+                {t('login')}
+              </LoadingButton>
+            </Grid>
+          </Grid>
+        </Box>
 
-            <Typography
-              gutterBottom
-              variant="body1"
-              sx={{
-                color: '#d32f2f',
-              }}
-            >
-              {error}
-            </Typography>
-          </>
-        )}
-
-        {isLoggedIn && (
-          <Button variant="outlined" onClick={logout}>
-            {'Déconnexion'}
-          </Button>
-        )}
+        <Typography
```

Comment:
create a re-usable component name `ErrorMessage` that defines a typography component to use when you need to display error messages